src/routes/admin/profiles/edit.test.ts

```diff
@@ -350,6 +351,79 @@ test('test handleGenerateRandomMEBxPassword when the request body generateRandom
   expect(result).toEqual(expected)
 })
 
+describe('handleWifiConfigs tests', () => {
+  let newConfig: AMTConfiguration
+  let oldConfig: AMTConfiguration
+
+  const profileWifiConfigsDb = {
+    deleteProfileWifiConfigs: jest.fn().mockReturnValue(null),
```

Comment:
I think these mocks  should match the return signatures accurately: 
Promise<boolean> and Promise<ProfileWifiConfigs[]>